### PR TITLE
chip: add an ellipsis menu for additional actions

### PR DESCRIPTION
**Pull request by Kiarokh**
See the description in the linked issue

fix https://github.com/Lundalogik/crm-feature/issues/4023

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Comment by adrianschmidt** [comment]
Please update the PR title to something that describes all the changes instead of just one part 🙏

**Comment by Kiarokh** [comment]
> Please update the PR title to something that describes all the changes instead of just one part 🙏

dude, I just made the PR 😄 wait a sec

**Review by Befkadu1 — APPROVED** [review]

**Comment by Befkadu1** [comment]
Maintainers needed here @jgroth or @adrianschmidt 🙏

File: etc/lime-elements.api.md [hunk 1]
```diff
@@ -2121,6 +2126,8 @@ export interface MenuItem<T = any> {
     text: string;
     value?: T;
 }
+export { MenuItem }
+export { MenuItem as MenuItem1 }
```
Comment (adrianschmidt) [inline etc/lime-elements.api.md:2130]: I don't understand why api-extractor suddenly felt the need to export this interface twice, but I don't think it causes any problems… 🤷‍♂️

**Review by adrianschmidt — APPROVED** [review]
Changes to the external API approved.

**Pull request merged**